BassBotDotNET/SlashCommandModules/FeedbackSlashCommandModule.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Discord.Net;
+using Discord;
+using Newtonsoft.Json;
+using Discord.WebSocket;
+
+namespace BassBotDotNET.SlashCommandModules;
+
+public class FeedbackSlashCommandModule
+{
+
+    /// <summary>
+    /// The feedback command with preset rating options.
+    /// </summary>
+    /// <returns>The command builder to use.</returns>
+    public SlashCommandBuilder Command()
+    {
+        var guildCommand = new SlashCommandBuilder()
+            .WithName("feedback")
+            .WithDescription("Tell us how much you are enjoying this bot!")
+            .AddOption(new SlashCommandOptionBuilder()
+                .WithName("rating")
+                .WithDescription("The rating your willing to give our bot")
```

Comment:
Very minor comment: Use "you're" instead of "your."

Reply:
thanks! shame on the discord.net bot documentation for this error.